gateway/1.4/modules/ROOT/pages/_partials/prerequisites.adoc

```diff
@@ -38,3 +38,12 @@ Red Hat Enterprise Linux (8, 9)
 //tag::ubuntu[]
 Ubuntu (Xenial, Bionic, Focal, Jammy)
 //end::ubuntu[]
+
```

Comment:
below is related to [W-13114488](https://gus.lightning.force.com/a07EE00001QJfPnYAL)
needed for openshift and other k8 deployments where rback is enabled